floors/fetcher.go

```diff
@@ -0,0 +1,323 @@
+package floors
+
+import (
+	"container/heap"
+	"context"
+	"encoding/json"
+	"errors"
+	"io"
+	"math"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/alitto/pond"
+	validator "github.com/asaskevich/govalidator"
+	"github.com/coocood/freecache"
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/metrics"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/prebid/prebid-server/util/timeutil"
+)
+
+var refetchCheckInterval = 300
+
+type FetchInfo struct {
+	config.AccountFloorFetch
```

Comment:
For my own understanding, why does this line not have a variable name attached to it?

Reply:
This is Struct embedding in Golang
With this we can inherit method which are defined on the internal struct. Also we can directly access the attributes inside internal struct without calling internal struct name explicitly

For more details you can refer
[Embedding](https://go.dev/doc/effective_go#embedding)
https://gobyexample.com/struct-embedding